terminado/websocket.py

```diff
@@ -56,6 +63,27 @@ def initialize(self, term_manager):
 
         self._logger = logging.getLogger(__name__)
 
+        self.keep_alive_time = keep_alive_time #send a ping if there is no activity for more than this many of seconds
```

Comment:
Code formatting time - I reckon this line is quite a lot longer than 80 characters. Can you put the comment on a separate line above?